modules/lib/maintainers.nix

```diff
@@ -370,6 +370,12 @@
     github = "mainrs";
     githubId = 5113257;
   };
+  nyarly = {
+    name = "Judson";
+    email = "nyarly@gmail.com";
+    github = "nyarly";
+    githubId = 127548;
+  };
```

Comment:
No need to add yourself here since you are already in the Nixpkgs maintainers list.